### DISCUSSION TITLE
The new feed feels cold, impersonal, and unpredicable 😢

### DISCUSSION THREAD

**Discussion by brendanzab** [community/community#66223] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I used to be able to see what people I followed were starring, who they were following etc. in a predicable list. I used to be able to star an obscure repository and see and immediate boost to the repository, with familiar faces starring it too. I don’t see how that can ever happen again under this new system.[^1] 😢 💔

It feels like my connection to those familiar faces I used to see in my homepage have been silenced. Now I just have a bunch of random events and repository updates from things that I don’t care about in the slightest, with no connection to anything or anyone I remember. It seems I can “train” it to see less of certain things, but that doesn’t change the fact that it feels very… cold, disconnected, unpredictable, and desolate.

I understand why a feed of repository releases etc. might be useful, but please don’t force it as a default (perhaps put it under another tab?), and _please_ make it predicable and non-algorithmic. I really hope you go back to the drawing board on this and let us use the old feed by default for the time being. It was far from perfect, but there was goodness there that I feel has been squandered.

[^1]: I really fear it will result in the most popular projects and repositories forever staying at the top, with no chance for smaller, more obscure projects getting a chance to get noticed

**Comment by conquera99** [comment]
yes, i need to view push activity from my own repository, but it not showing

I set the filter to repo activity only, but its the same

**Comment by ghost** [comment]
https://github.com/orgs/community/discussions/66188
